001-core/privacy.php

```diff
@@ -465,7 +465,7 @@ function delete_old_export_files() {
  * @return string The modified robots.txt content.
  */
 function vip_convenience_domain_robots_txt( $output ) {
-	$host = strtolower( $_SERVER['HTTP_HOST'] ?? '' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	$host = $_SERVER['HTTP_HOST'] ?? ''; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
why do we drop `strtolower` here?

Reply:
Because it's already checked @ https://github.com/Automattic/vip-go-mu-plugins/pull/5130. I need to merge that one first tho